specs/experimental/plasma.md

```diff
@@ -83,6 +83,17 @@ on the `commitment_type_byte` where [0, 128) are reserved for official implement
 | `commitment_type` | `commitment`                    |
 | ----------------- | ------------------------------- |
 | 0                 | `keccak256(tx_payload)`         |
+| 1                 | `da-service`                     |
+
+The `da-service` commitment is as follows: `da_layer_byte ++ len(payload) as uvarint ++ payload`. The DA layer byte
+must be initially restricted to the range `[0, 127)`. This specification will not apportion DA layer bytes, but
+different DA layers should coordinate to ensure that the DA layer bytes do not conflict. The payload length is a
```

Comment:
How will DA layers be able to coordinate such that they do not conflict? I lean towards either having a table in this doc or linking someplace where it can be discussed, ie a github issue in this repo

Reply:
I added a link to a github discussion thread